src/CodeGen_C.cpp

```diff
@@ -152,15 +152,39 @@ inline float float_from_bits(uint32_t bits) {
 }
 
 template<typename T>
-inline int halide_popcount(T a) {
+inline int halide_popcount_fallback(T a) {
```

Comment:
I would leave a link to the source for this algorithm for the reference (found this one https://graphics.stanford.edu/~seander/bithacks.html#CountBitsSetKernighan, for example).